server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -293,3 +293,12 @@ Feature: GetMessages method
     And the list should contain 1 message
     And the preview of the message is not empty
     And the preview should not contain consecutive spaces or blank characters
+
+  Scenario: Preview should respect with iso charset
```

Comment:
I think the scenario name can be improved...

What do we respect? Is respect an appropriate verb here?